src/append/opentelemetry.rs

```diff
@@ -174,19 +174,16 @@ impl OpentelemetryLogBuilder {
             layout,
         } = self;
 
-        let collector_timeout = opentelemetry_otlp::OTEL_EXPORTER_OTLP_TIMEOUT_DEFAULT;
         let exporter = match protocol {
             Protocol::Grpc => LogExporter::builder()
                 .with_tonic()
                 .with_endpoint(endpoint)
                 .with_protocol(protocol)
-                .with_timeout(collector_timeout)
```

Comment:
Default to `OTEL_EXPORTER_OTLP_TIMEOUT_DEFAULT` and respect the logic load from environment variable.